com.unity.multiplayer.mlapi/Runtime/Messaging/InternalMessageHandler.cs

```diff
@@ -342,6 +342,22 @@ void DelayedSpawnAction(Stream continuationStream)
                             }
                         }
 
+                        // Clean up the diffed scene objects. I.E scene objects that have been destroyed
+                        if (SpawnManager.pendingSoftSyncObjects.Count > 0)
```

Comment:
Could this be hoisted into a function inside SpawnManager?  There are other functions like this e.g. DestroySceneObjects

Reply:
Ok, just pushed a commit to refactor this a bit